src/content/docs/ra-primer/relations.mdx

```diff
@@ -73,7 +73,7 @@ Likewise, the Cartesian product of the three sets **People**, **Garments**, and
 Tables in relational databases are extensional definitions of relations: they list the tuples for which a certain relation holds. Views in relational databases are intensional definitions of relations: they define new relations by introducing additional predicates on top of already defined tables (or other views).
 
 :::note
-From a purist point of view, it is more correct to name database tables eg `user` rather than `users`, because a table is **a relation** rather than a list of things. But then again, an extensional definition of the `user` relation is precisely... a list of users.
+From a purist point of view, it is more correct to name database tables eg `user` rather than `users`, because a table is **a relation** rather than a list of things. But then again, an extensional definition of the `user` relation is precisely... a set of users.
```

Comment:
This is meant to be informal. The point is that *if we are not purist* about it, we can think of a relation as a list of Xs. I think we make the point that relations can't have duplicates pretty strongly in other places, so a more relaxed formulation here seems ok to me.

Reply:
Disagree. The sentence starts with "From a purist point of view ..." and ends up saying" a relation is **precisely** a list", which is very exactly the kind of shortcuts Date & Darwen fight for decades now. 

At least remove **purist** and **precisely**.